entries/r/rb.cz.json

```diff
@@ -1,11 +1,12 @@
 {
   "Raiffeisen Bank [CZ]": {
     "domain": "rb.cz",
-    "url": "https://www.rb.cz/",
-    "tfa": [
-      "custom-hardware"
-    ],
-    "documentation": "https://www.rb.cz/firemni-finance/podnikatele-a-male-firmy/prime-bankovnictvi/sluzby-pro-firemni-ucty/bezpecnost-internetoveho-bankovnictvi/",
```

Comment:
Their documentation states that this is only used to confirm certain actions, and not for accessing the account.